ipaserver/plugins/idviews.py

```diff
@@ -790,8 +791,11 @@ def pre_callback(self, ldap, dn, *keys, **options):
         except errors.NotFound:
             self.obj.handle_not_found(*keys)
 
-        required_object_classes = set(self.obj.object_class)
-        actual_object_classes = set(entry['objectclass'])
+        def __lower(text):
+            return text.lower()
+
+        required_object_classes = set(map(__lower, self.obj.object_class))
+        actual_object_classes = set(map(__lower, entry['objectclass']))
```

Comment:
Hi @felipevolpone 
For consistency, you can use the same method as in line 550:

for required_oc in required_object_classes:
    if not self.obj.has_objectclass(entry['objectclass'], required_oc):
        ....

Reply:
Thanks for pointing me that :)